include/switcher.php

```diff
@@ -122,12 +122,13 @@ protected function get_elements( $args ) {
 		$out   = array();
 
 		foreach ( $this->links->model->get_languages_list( array( 'hide_empty' => $args['hide_if_empty'] ) ) as $language ) {
-			$id = (int) $language->term_id;
-			$order = (int) $language->term_group;
-			$slug = $language->slug;
-			$locale = $language->get_locale( 'display' );
+			$id           = (int) $language->term_id;
+			$order        = (int) $language->term_group;
+			$slug         = $language->slug;
+			$locale       = $language->get_locale( 'display' );
+			$w3c          = $language->get_locale( 'display' );
```

Comment:
It's a bit strange to add a 2nd property with the same value as an existing one. Why?
If I well understood the issue, it's not here that it should be fixed.

Reply:
Since we're getting the `w3c` [here](https://github.com/polylang/polylang/blob/c4fcd02aa52cb40eedfec8d2bc512c68e275140b/include/walker-dropdown.php#L39), we have to set it.